contracts/MultiPartyEscrow.sol

```diff
@@ -31,11 +32,14 @@ contract MultiPartyEscrow {
  
     ERC20 public token; // Address of token contract
     
-    //TODO: optimize events. Do we need more (or less) events?
-    event EventChannelOpen       (uint256 channelId,         address indexed sender, address indexed recipient, uint256 indexed groupId);
-    //event EventChannelReopen     (uint256 channelId,         address indexed sender, address indexed recipient, uint256 indexed groupId, uint256 nonce);
-    //event EventChannelTorecipient(uint256 indexed channelId, address indexed sender, address indexed recipient, uint256 amount);
-    //event EventChannelTosender   (uint256 indexed channelId, address indexed sender, address indexed recipient, uint256 amount);
+
+    // Events
+    event ChannelOpen(uint256 channelId, address indexed sender, address indexed recipient, uint256 indexed groupId, address signer, uint256 amount, uint256 expiration);
+    event ChannelClaim(address indexed recipient, uint256 channelId, uint256 claimAmount, uint256 sendBackAmount, uint256 keepAmpount);
+    event SenderClaim(uint256 channelId, uint256 claimAmount);
+    event ExtendChannel(uint256 channelId, uint256 newExpiration);
```

Comment:
I would suggest using `Channel` in a consistent way

eg. `ChannelOpen` != `ExtendChannel`. Would it be better `ChannelExtend` instead?